src/localisation/string_ids.h

```diff
@@ -2303,7 +2303,7 @@ enum {
 	STR_ACTION_NA = 5662,
 	STR_ACTION_CLEAR_LANDSCAPE = 5663,
 	STR_ACTION_CHEAT = 5664,
-	
+	STR_ACTION_TOGGLE_SCENERY_CLUSTER = 5665,
```

Comment:
You should probably keep an empty line here since it was made to divide the constants